zookeeper-server/src/test/java/org/apache/zookeeper/server/NettyServerCnxnTest.java

```diff
@@ -85,10 +85,9 @@ public void tearDown() throws Exception {
     public void testSendCloseSession() throws Exception {
         assertTrue("Didn't instantiate ServerCnxnFactory with NettyServerCnxnFactory!", serverFactory instanceof NettyServerCnxnFactory);
 
-        final ZooKeeper zk = createClient();
-        final ZooKeeperServer zkServer = serverFactory.getZooKeeperServer();
-        final String path = "/a";
-        try {
+        try (ZooKeeper zk = createClient()) {
+            final ZooKeeperServer zkServer = serverFactory.getZooKeeperServer();
```

Comment:
Please don't do unrelated refactor work next time

Reply:
Sorry for that. I will keep in mind.